dart/constraint/ContactConstraint.hpp

```diff
@@ -185,6 +208,12 @@ class ContactConstraint : public ConstraintBase
   /// Primary Coefficient of Friction
   double mSecondaryFrictionCoeff;
 
+  /// Primary Coefficient of Slip Compliance
+  double mSlipCompliance;
```

Comment:
Nit: Could you rename this to `mPrimarySlipCompliance` to be consistent with the friction coefficient variables?

Reply:
Done in https://github.com/dartsim/dart/pull/1505/commits/5d81fbbc3a61d493f7f4f201657aa482ed169d2c.